mapadroid/db/madmin_conversion.py

```diff
@@ -81,7 +81,6 @@
         `post_pogo_start_delay` float DEFAULT NULL,
         `restart_pogo` int(11) DEFAULT NULL,
         `inventory_clear_rounds` int(11) DEFAULT NULL,
-        `inventory_clear_item_amount_tap_duration` int(11) DEFAULT NULL,

```

Comment:
I'd assume adjusting `scripts/SQL/rocketmap.sql` would also be needed

Reply:
Also done. Thanks 👍